qa/workunits/rbd/rbd_mirror.sh

```diff
@@ -7,12 +7,7 @@
 # socket, temporary files, and launches rbd-mirror daemon.
 #
 
-if [ -n "${CEPH_REF}" ]; then
-  wget -O rbd_mirror_helpers.sh "https://git.ceph.com/?p=ceph.git;a=blob_plain;hb=$CEPH_REF;f=qa/workunits/rbd/rbd_mirror_helpers.sh"
-  . ./rbd_mirror_helpers.sh
-else
-  . $(dirname $0)/rbd_mirror_helpers.sh
-fi
+. ./rbd_mirror_helpers.sh
```

Comment:
@liewegas I suppose it wont work when running this locally from cmake build dir, i.e.:
```
../qa/workunits/rbd/rbd_mirror.sh
```